public/app/features/dashboard/timepicker/timepicker.ts

```diff
@@ -118,6 +117,7 @@ export class TimePickerCtrl {
   }
```

Comment:
Here we probably want to emit appEvent that timepicker is opened, i.e. `timepicker-opened`.